src/app/amsterdam/bestuur-en-organisatie/college-van-burgemeester-en-wethouders/page.tsx

```diff
@@ -0,0 +1,130 @@
+
+import municipalExecutive from '../../_assets/municipal-executive.jpg'
+import municipalExecutiveDeJager from '../../_assets/municipal-executive-de-jager.jpg'
+import municipalExecutiveGrootWassink from '../../_assets/municipal-executive-groot-wassink.jpg'
+import municipalExecutiveHalsema from '../../_assets/municipal-executive-halsema.jpg'
+import municipalExecutiveMbarki from '../../_assets/municipal-executive-mbarki.jpg'
+import municipalExecutiveMeliani from '../../_assets/municipal-executive-meliani.jpg'
+import municipalExecutiveMoorman from '../../_assets/municipal-executive-moorman.jpg'
+import municipalExecutivePels from '../../_assets/municipal-executive-pels.jpg'
+import municipalExecutiveScholtes from '../../_assets/municipal-executive-scholtes.jpg'
+import municipalExecutiveTeesink from '../../_assets/municipal-executive-teesink.jpg'
+import municipalExecutiveVanBuren from '../../_assets/municipal-executive-van-buren.jpg'
+import municipalExecutiveVanDantzig from '../../_assets/municipal-executive-van-dantzig.jpg'
+import municipalExecutiveVanDerHorst from '../../_assets/municipal-executive-van-der-horst.jpg'
+
+const persons: Array<PersonProps> = [
+  {
+    fullName: 'Femke Halsema',
+    image: municipalExecutiveHalsema,
+    jobTitle: 'Burgemeester',
+  },
+  {
+    fullName: 'Marjolein Moorman',
+    image: municipalExecutiveMoorman,
+  },
+  {
+    fullName: 'Rutger Groot Wassink',
+    image: municipalExecutiveGrootWassink,
+  },
+  {
+    fullName: 'Reinier van Dantzig',
+    image: municipalExecutiveVanDantzig,
+  },
+  {
+    fullName: 'Sofyan Mbarki',
+    image: municipalExecutiveMbarki,
+  },
+  {
+    fullName: 'Touria Meliani',
+    image: municipalExecutiveMeliani,
+  },
+  {
+    fullName: 'Melanie van der Horst',
+    image: municipalExecutiveVanDerHorst,
+  },
+  {
+    fullName: 'Hester van Buren',
+    image: municipalExecutiveVanBuren,
+  },
+  {
+    fullName: 'Zita Pels',
+    image: municipalExecutivePels,
+  },
+  {
+    fullName: 'Alexander Scholtes',
+    image: municipalExecutiveScholtes,
+  },
+  {
+    fullName: 'Dirk de Jager',
+    image: municipalExecutiveDeJager,
+  },
+  {
+    fullName: 'Peter Teesink',
+    image: municipalExecutiveTeesink,
+    jobTitle: 'Gemeentesecretaris',
+  },
+]
+
+type PersonProps = {
+  fullName: string
+  image: StaticImageData
+  jobTitle?: string
+}
+
+function PersonCardCell({ fullName, image, index, jobTitle }: PersonProps & { index: number }) {
+  return (
+    <Grid.Cell span={3} start={{ narrow: 1, medium: index % 2 === 0 ? 2 : 5, wide: (index % 4) * 3 + 1 }}>
+      <Card>
+        <AspectRatio ratio="x-wide">
+          <NextImage alt="" src={image} fill />
+        </AspectRatio>
+        <Heading size="level-4">
+          <Card.Link href="#">{fullName}</Card.Link>
+        </Heading>
+        <Paragraph>{jobTitle ?? 'Wethouder'}</Paragraph>
+      </Card>
+    </Grid.Cell>
+  )
+}
+
+PersonCardCell.defaultProps = {
+  jobTitle: 'Wethouder',
+}
```

Comment:
@alimpens Did I do this goodly?

Reply:
Yes. And me too, but we have an ESLint rule against removing `defaultProps`:

<img width="804" alt="Screenshot 2024-12-17 at 10 21 04" src="https://github.com/user-attachments/assets/caaa7e17-f6d4-44bb-b690-802772dcad53" />